releasenotes/notes/support-32-bit-address-spaces-5b5b36c2d9f9ff00.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+----
+fixes:
+  - |
+    Support building on 32bit armv7
```

Comment:
just to be extra-clear regarding our support policy, could you reword this to avoid saying we `support` the 32bit arm build? (something like `Fix build failure on 32bit armv7` would do) :)

Reply:
Done